mypy.ini

```diff
@@ -17,5 +17,11 @@ disallow_any_generics=False
 warn_unreachable=True
 
 # We will add more files over time to increase coverage
-; files = **/modin/*.py
-files = **/modin/config/*.py, **/modin/logging/*.py
+; files = modin/*.py
+files = 
+        modin/config/*.py,
+        modin/logging/*.py,
+        modin/_version.py,
+        modin/db_conn.py,
+        modin/error_message.py,
+        modin/utils.py
```

Comment:
Any reason why we're skipping `apply_license_header.py` so that we can just do `modin/*.py` here?

Reply:
I only annotated files that get imported elsewhere in the codebaes, and `modin/conftest.py` shows a lot of errors for missing annotations. I can make mypy ignore conftest.py and add simple annotations for main.py and __init__.py to make it easier.